css/_innovation-landing.scss

```diff
@@ -91,11 +96,17 @@ $font-path: '../../fonts';
         margin-top: $layout-lg;
 
         .mzp-c-picto {
-            height: 100%;
 
             p {
                 @include font-size(16px);
             }
+
+            > div {
+                display: flex;
+                flex-direction: column;
+                height: 100%;
+                justify-content: space-evenly;
```

Comment:
**Nit:** Change this property to be `justify-content: space-evenly`, since it's not aligning the icons to the top, and creating more uneven spaces between the content.

**w/ `space-evently`**:
<kbd>![Screen Shot 2023-03-09 at 2 28 33 PM](https://user-images.githubusercontent.com/42309026/223909227-17323ed0-43ef-47ee-9a78-bccf8f73b954.png)</kbd>

**w/ `space-between`:
<kbd>![Screen Shot 2023-03-09 at 2 29 37 PM](https://user-images.githubusercontent.com/42309026/223909383-5586855a-60bb-4aac-97dc-a5f9350a668b.png)</kbd>